src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -2589,6 +2626,12 @@ protected void handleCDPLoad(boolean allowedToLoadConfig) {
                     
                     File cdpConfigDir = loader.findCDPConfigDirectory(new File(entryLocation));
                     
+                    if (contest.getJudgements().length == 0) {
+                        if (loadJudgements(contest, cdpConfigDir)){
+                            System.out.println("Loaded judgements from "+getCdpConfigRejectIniFilename(cdpConfigDir));
+                        }
+                    }
+                    
```

Comment:
Ditto the previous review comments; this does not appear to have anything to do with Issue #119 or PR #120.